bitnami/minio/values.yaml

```diff
@@ -274,6 +274,7 @@ provisioning:
   ##     region: us-east-1
   ##     # Only when mode is 'distributed'
   ##     # ref: https://docs.minio.io/docs/distributed-minio-quickstart-guide
+  ##     versioning: false
```

Comment:
It could be nice to add a link to the `version` command at the top of this block, where the links to the other (`mb`, `ilm`, `bucket`...) commands are. Something like this:

```
## Versioning https://docs.min.io/docs/minio-client-complete-guide.html#version
```

Reply:
Ive added a link to the versioning guide

Reply:
Can you also add the link I suggested around lines 267-270?